conda/recipes/librmm/recipe.yaml

```
@@ -0,0 +1,115 @@
+# Copyright (c) 2018-2025, NVIDIA CORPORATION.
+schema_version: 1
+
+
+# bashlet to use for this:
+# IFS=- read -r RAW_VERSION GIT_DESCRIBE_NUMBER GIT_DESCRIBE_HASH <<< $(git describe --tags)
+# Takes string like "v25.02.00a-49-g77dadb88" and splits on the `-` and drops the result into three variables
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
+  cuda_version: ${{ (env.get('RAPIDS_CUDA_VERSION') | split('.'))[:2] | join(".") }}
+  cuda_major: ${{ (env.get('RAPIDS_CUDA_VERSION') | split('.'))[0] }}
+  date_string: ${{ env.get("RAPIDS_DATE_STRING") }}
+  GIT_DESCRIBE_HASH: ${{ env.get("GIT_DESCRIBE_HASH") }}
+  GIT_DESCRIBE_NUMBER: ${{ env.get("GIT_DESCRIBE_NUMBER") }}
+
+recipe:
+  name: librmm-split
+
+cache:
+  source:
+    path: ../../..
+
+  requirements:
+    build:
+      - cmake ${{ cmake_version }}
+      - ninja
+      - make
+      - ${{ compiler('c') }}
+      - ${{ compiler('cxx') }}
+      - ${{ compiler('cuda') }} =${{ cuda_version }}
+      - cuda-version =${{ cuda_version }}
```

The `cuda-version` constraint should already constrain the CUDA compiler version. So think we can drop the constraint from the former

```suggestion
      - ${{ compiler('cuda') }}
      - cuda-version =${{ cuda_version }
```